src/store/ConfigStore.ts

```diff
@@ -13,12 +22,12 @@ const configureStore = (initialState?: KialiAppState) => {
   const middlewares = [thunk];
   // compose enhancers
   const enhancer = composeEnhancers(applyMiddleware(...middlewares));
+  // persist reducers
+  const pReducer = persistReducer(persistConfig, rootReducer);
   // create store
-  return createStore(rootReducer, initialState, enhancer);
+  return createStore(pReducer, initialState, enhancer);
```

Comment:
Can we spell out `persistentReducer` instead of `pReducer`? I don't know what a `pReducer` is without looking around in the code and see how the object was instantiated. No need to be terse, meaningful names is   key. Besides, in production it will get minified anyway.

Reply:
Changed @mtho11 Thanks